setup.py

```diff
@@ -4,7 +4,7 @@
 
 install_requires = [
     'PyYAML',
-    'charmhelpers'
+    'charms.benchmark'
```

Comment:
We should hard-code the version here

```
    'charms.benchmark>=1.0<1.1'
```

That way in charms.benchmark 1.1 we won't potentially break the nearly 1:1 copy of charm-benchmark 1.1.1 to charms.benchmark 1.0.1